docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,176 @@
+
+### Catalog properties
+
+You can pass to a Doris data source any property that isn't defined by Gravitino by adding
+`gravitino.bypass.` prefix as a catalog property. For example, catalog property
+`gravitino.bypass.maxWaitMillis` will pass `maxWaitMillis` to the data source property.
+
+You can check the relevant data source configuration in
+[data source properties](https://commons.apache.org/proper/commons-dbcp/configuration.html) for
+more details.
+
+Here are the catalog properties defined in Gravitino for Doris catalog:
+
+| Configuration item   | Description                                                                         | Default value | Required | Since Version |
+|----------------------|-------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:9030` | (none)        | Yes      | 0.5.0         |
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver`.            | (none)        | Yes      | 0.5.0         |
+| `jdbc-user`          | The JDBC user name.                                                                 | (none)        | Yes      | 0.5.0         |
+| `jdbc-password`      | The JDBC password.                                                                  | (none)        | Yes      | 0.5.0         |
+| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                      | `2`           | No       | 0.5.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                     | `10`          | No       | 0.5.0         |
+
+Before using the Doris Catalog, you must download the corresponding JDBC driver to the `catalogs/jdbc-doris/libs` directory.
+Gravitino doesn't package the JDBC driver for Doris due to licensing issues.
+
+### Catalog operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#catalog-operations) for more details.
+
+## Schema
+
+### Schema capabilities
+
+- Gravitino's schema concept corresponds to the Doris database.
+- Supports creating schema.
+- Supports dropping schema.
+
+### Schema properties
+
+- Support schema properties, includes Doris database properties and user-defined properties.
+
+### Schema operations
+
+Please refer to
+[Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#schema-operations) for more details.
+
+## Table
+
+### Table capabilities
+
+- Gravitino's table concept corresponds to the Doris table.
+- Supports index.
+- Supports [column default value](./manage-relational-metadata-using-gravitino.md#table-column-default-value) and [auto-increment](./manage-relational-metadata-using-gravitino.md#table-column-auto-increment)..
```

Comment:
You mentioned that we don't support auto-increment for now in the below, but here it says it supports auto-increment, so which one is correct? @zhoukangcn @yuqi1129

Reply:
Sorry, It's a mistake here. I will update doc today